config/routes.rb

```diff
@@ -10,14 +10,16 @@
 
   namespace :api do
     namespace :v1 do 
-      resources :sauces, only: [:show]
+      resources :sauces, only: [:show] do 
+        resources :reviews, only: [:show, :create]
+      end 
       resources :users, only: [:show]
     end 
   end 
 
-  namespace :api do
-    namespace :v1 do 
-      resources :reviews, only: [:show]
-    end 
-  end 
+ 
+  #   namespace :v1 do 
+  #     resources :reviews, only: [:show]
+  #   end 
+  # end 
```

Comment:
um....lets delete this if it isnt being used

Reply:
Deleted unused code